wpilib/wpilib/adxl362.py

```diff
@@ -43,19 +46,20 @@ class ADXL362(SensorBase):
 
     Range = Accelerometer.Range
 
-    class Axes:
+    class Axes(IntEnum):
         kX = 0x00
         kY = 0x02
         kZ = 0x04
 
-    def __init__(self, range, port=None):
+    def __init__(self, range: Range, port=None):
         """Constructor.
         
         :param range: The range (+ or -) that the accelerometer will measure.
         :type range: :class:`.ADXL362.Range`
         :param port: The SPI port that the accelerometer is connected to
         :type port: :class:`.SPI.Port`
         """
+        super().__init__()
```

Comment:
Yeah.. since you keep adding them, let's pull those out and make them a separate PR. I agree the change should be made, it just needs to be added separately I think.